nextjs-blog/components/speeddial.js

```diff
@@ -0,0 +1,54 @@
+import * as React from "react";
+import { useState } from "react";
+import Box from "@mui/material/Box";
+import FullscreenOverlay from "../components/transoverlay";
+import SpeedDial from "@mui/material/SpeedDial";
+import SpeedDialIcon from "@mui/material/SpeedDialIcon";
```

Comment:
Again, just for knowledge we could import these as follows

`import { SpeedDial, SpeedDialIcon } from "@mui/material"`

Reply:
Acknowledged and adjusted :D